Azure-Samples/azure-search-openai-demo
# add page to upload files

## Purpose

Add page to upload PDF files directly from the UI instead of running separate script

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
azd up
cd app
run start.cmd or start.sh
go to http://127.0.0.1:5000/#/ul
upload the PDF files from the repository data directory
```

* Test the code

Manual test
```
check console for success of file upload and index status
```

## What to Check
Verify that the following are valid:
* check the chat and  ask your data is working as expected

## Other Information

UI page could make use of status message better (need help on react code to update progress for long running jobs)
